mmv1/third_party/terraform/resources/resource_bigtable_table.go

```diff
@@ -27,7 +27,8 @@ func ResourceBigtableTable() *schema.Resource {
 
 		// Set a longer timeout for table creation as adding column families can be slow.
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(45 * time.Minute),
```

Comment:
Is there a reason for lowering the default create timeout? As this is optional I feel it would make sense to keep it at the currently Extended value if that has proved adequate until now.

Reply:
Fixed.